src/frontend/org/voltdb/VoltDB.java

```
@@ -665,20 +668,34 @@ else if (arg.equalsIgnoreCase("force")) {
                 } else if (arg.equalsIgnoreCase("forceget")) {
                     m_forceGetCreate = true;
-                } else if (arg.equalsIgnoreCase("schema")){
+                } else if (arg.equalsIgnoreCase("schema")) {
                     m_userSchema = new File(args[++i].trim());
-                    if (!m_userSchema.exists()){
+                    if (!m_userSchema.exists()) {
                         System.err.println("FATAL: Supplied schema file " + m_userSchema + " does not exist.");
                         referToDocAndExit();
                     }
-                    if (!m_userSchema.canRead()){
+                    if (!m_userSchema.canRead()) {
                         System.err.println("FATAL: Supplied schema file " + m_userSchema + " can't be read.");
                         referToDocAndExit();
                     }
-                    if (!m_userSchema.isFile()){
+                    if (!m_userSchema.isFile()) {
                         System.err.println("FATAL: Supplied schema file " + m_userSchema + " is not an ordinary file.");
                         referToDocAndExit();
                     }
+                } else if (arg.equalsIgnoreCase("classes")) {
+                    m_stagedClassesPath = new File(args[++i].trim());
+                    if (!m_stagedClassesPath.exists()){
+                        System.err.println("FATAL: Supplied classes jar file " + m_stagedClassesPath + " does not exist.");
+                        referToDocAndExit();
+                    }
+                    if (!m_stagedClassesPath.canRead()) {
+                        System.err.println("FATAL: Supplied classes jar file " + m_stagedClassesPath + " can't be read.");
+                        referToDocAndExit();
+                    }
+                    if (!m_stagedClassesPath.isFile()) {
+                        System.err.println("FATAL: Supplied classes jar file " + m_stagedClassesPath + " is not an ordinary file.");
```

It looks like all the argument parsing errors are FATAL and result in Volt not starting. I'm open to discussing this with Andrew, but as a user, I'd rather have the command fail than succeed with unexpected behavior.